pkg/certificate/types.go

```diff
@@ -0,0 +1,21 @@
+package certificate
+
+import (
+	"fmt"
+)
+
+type WebhookType string
+
+const (
+	MutatingWebhook   WebhookType = "Mutating"
+	ValidatingWebhook WebhookType = "Validating"
+)
+
+type WebhookReference struct {
+	Type WebhookType
+	Name string
+}
+
+func (w WebhookReference) String() string {
+	return fmt.Sprintf("%s/%s", w.Type, w.Name)
```

Comment:
Let's add "Webhook" suffix in case we print this out of webhook context 

```suggestion
	return fmt.Sprintf("%sWebhook/%s", w.Type, w.Name)
```